apps/backoffice-v2/src/common/components/molecules/ProcessTracker/hooks/useProcessTrackerLogic/process-tracker-adapters/collection-flow.process-tracker.ts

```diff
@@ -0,0 +1,46 @@
+import { processStatusToIcon } from '@/common/components/molecules/ProcessTracker/constants';
+import {
+  IProcessTracker,
+  ProcessTrackerItem,
+} from '@/common/components/molecules/ProcessTracker/hooks/useProcessTrackerLogic/process-tracker-adapters/process-tracker.abstract';
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { ProcessStatus } from '@ballerine/common';
+import { titleCase } from 'string-ts';
+
+export class CollectionFlowProcessTracker implements IProcessTracker {
+  PROCESS_NAME = 'collection-flow';
+
+  constructor(readonly workflow: TWorkflowById) {}
+
+  buildItems(): ProcessTrackerItem[] {
+    return (
+      this.getSteps()?.map(step => {
+        return {
+          text: titleCase(step),
+          leftIcon: this.getCollectionFlowStatus(step),
+        };
+      }) || []
+    );
+  }
+
+  getReadableName(): string {
```

Comment:
Redundant return type.
```suggestion
  getReadableName() {
```